packages/docsearch/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@astrojs/starlight-docsearch",
+  "version": "0.0.0",
+  "description": "Algolia DocSearch plugin for the Starlight documentation theme for Astro",
+  "author": "Chris Swithinbank <swithinbank@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/withastro/starlight",
+    "directory": "packages/docsearch"
+  },
+  "bugs": "https://github.com/withastro/starlight/issues",
+  "homepage": "https://starlight.astro.build",
+  "type": "module",
+  "files": [
+    "index.ts",
+    "DocSearch.astro",
+    "schema.ts",
+    "variables.css",
+    "virtual.d.ts"
+  ],
+  "exports": {
+    ".": "./index.ts",
+    "./DocSearch.astro": "./DocSearch.astro",
+    "./schema": "./schema.ts"
+  },
+  "peerDependencies": {
+    "@astrojs/starlight": ">=0.9.0"
```

Comment:
Shouldn't this be way higher?

Reply:
Oh yeah 😅 

Copy/pasted from the Tailwind package, but will need v0.14. I’ll update once that’s released.

Reply:
Fixed in 77164f6